py/packages/genkit/src/genkit/ai/retriever.py

```diff
@@ -13,6 +13,11 @@
 from xml.dom.minidom import Document
 
 from genkit.core.typing import RetrieverResponse
+from pydantic import BaseModel, Field
 
 # User-provided retriever function that queries the datastore
 type RetrieverFn[T] = Callable[[Document, T], RetrieverResponse]
+
+
+class CommonRetrieverOptionsSchema(BaseModel):
```

Comment:
We probably don't want this anymore? There was some discussion on how useful these common types are.

@pavelgj

Reply:
Checked with Pavel, let us skip these common options schema.... they don't add much to the API.

Reply:
I moved it from retriever